include/crud-posts.php

```diff
@@ -324,4 +325,23 @@ public function create_media_translation( $post_id, $lang ) {
 		do_action( 'pll_translate_media', $post_id, $tr_id, $lang->slug );
 		return $tr_id;
 	}
+
+	/**
+	 * Ensure that tags are in the correct language when a post is updated, due to `tags_input` parameter being removed in `wp_update_post()`.
+	 *
+	 * @since 3.4.5
+	 *
+	 * @param int     $post_id      Post ID, unused.
+	 * @param WP_Post $post_after   Post object following the update.
+	 * @param WP_Post $post_before  Post object before the update.
+	 * @return void
+	 */
+	public function force_tags_translation( $post_id, $post_after, $post_before ) {
+		$post_before = $post_before->to_array();
+
+		if ( ! empty( $post_before['tags_input'] ) ) {
+			// Let's ensure that  `PLL_CRUD_Posts::set_object_terms()` will do its job.
+			wp_set_post_tags( $post_id, $post_before['tags_input'] );
```

Comment:
This may cause another problem. Please see my issue https://github.com/polylang/polylang/issues/1401